### ISSUE TITLE
Weekly assignments and events aren't working

### ISSUE THREAD

**Issue by edu8000** [0P3N50URC3-F0R3V3R/blackbeard#1]
I'm using your repack, followed all the instructions and managed to get deep dives working, no more red message on the deep dives console, but weekly assignments and events aren't working.

**Comment by 0P3N50URC3-F0R3V3R** [comment]
- Wich repack of mine are you using? What is the EXACT version number?
- Do you using my XAMPP or using a custom one?
- Did you checked the cert file and added to user and machine too?

If only the deep dives working and no others there is a port error...
You will need 443 and 8443... 

- Port 443 : - It uses Deepdive, and Weekly events and main event - So if only deepdives works it need to be work weekly challanges also...
- Port 8443 : - It uses Freebeer event and community triggers.

This issue can be caused by port or certificate error.

**Comment by edu8000** [comment]
I'm using this one: https://mega.nz/file/QAIXUCQI#k3TpGUQ5Iq2K5p2hPKqr6_nE2L7gFcYQK5sJMtFQQ7Q

I'm using the XAMPP included there, I've also installed the certificate like the tutorial said.

Free beer events are working, but not main events like christmas.

Is there any log file I can send you?

**Comment by 0P3N50URC3-F0R3V3R** [comment]
The system itself not using any log besides the game itself...(That is not logging this part) 
i cannot help in this way since i cannot reproduce the error...

What i could tell you that you should check the machine that runs xampp machine's firewall to let 443 and 8443 if that is not the same... Also try it if it works with 127.0.0.1 if you are running the web server on localhost. 

Other than this, i dont know since this problem never occoured to me...

**Comment by edu8000** [comment]
I also did like KalyaSc in the other post, with the other save I get week assignments but the big events like christmas aren't working.

**Comment by 0P3N50URC3-F0R3V3R** [comment]
This is strange. You do use Radmin too? Id you do please try without it... just on 127.0.0.1 in your hosts file, since for me on a LAN and my own Softether vpn everything works... :\

**Comment by edu8000** [comment]
You're right, weekly assignments won't show up until I'm player rank 8, that's why it works with the other save, KalyaSc problem should be fixed.

But events still don't work for me, I've tried with 127.0.0.1 and my own IP, I've also tried with the newer events.

**Comment by 0P3N50URC3-F0R3V3R** [comment]
IDK what is going on at you, if you check KaylaSc post, tzhat is working to him so i am not sure what is the problem at you...

**Comment by edu8000** [comment]
I finally managed to make it work, I looked at the events file and when selecting christmas, it said {"ActiveEvents":["Xmas"]}, I changed it to "{"ActiveEvents":["Xmas2021"]}" and it all works fine, the thing is, when selecting halloween 2021, It leaves the file blank, but when selecting the previous option, steam awards 2020, it says:
{"ActiveEvents":["SteamAwards2020"]}
{"ActiveEvents":["Halloween2021"]} 
So I just deleted the first line and It also works.

The problem seems to be the options on the blackbeard program itself not editing the events file correctly, I'm using the 0.4 version.

Problem solved, thanks for your time and for creating this program.
